packages/turbo-types/schemas/schema.json

```diff
@@ -223,6 +223,14 @@
           "type": "number",
           "description": "Sets a timeout for remote cache uploads. Value is given in seconds and only whole values are accepted. If `0` is passed, then there is no timeout for any remote cache uploads.",
           "default": 60
+        },
+        "teamId": {
+          "type": "string",
+          "description": "The team id of the Remote Cache team. Value will be passed as `teamId` in the querystring for all Remote Cache HTTP calls. Must start with `team_` or it will not be used."
```

Comment:
```suggestion
          "description": "The ID of the Remote Cache team. Value will be passed as `teamId` in the querystring for all Remote Cache HTTP calls. Must start with `team_` or it will not be used."
```